quarkus.ls/com.redhat.quarkus.ls/src/test/java/com/redhat/quarkus/services/ApplicationPropertiesCompletionTest.java

```diff
@@ -140,4 +145,28 @@ public void completionAfterProfil() throws BadLocationException {
 		testCompletionFor(value, true, c("quarkus.http.cors", "%dev.quarkus.http.cors=${1|false,true|}", r(0, 0, 5)));
 	}
 
+	@Test
+	public void noCompletionForExistingProperties() throws BadLocationException {
+
+		String value = "|";
+		
+		QuarkusProjectInfo projectInfo = new QuarkusProjectInfo();
+		List<ExtendedConfigDescriptionBuildItem> properties = new ArrayList<ExtendedConfigDescriptionBuildItem>();
+		ExtendedConfigDescriptionBuildItem p1 = new ExtendedConfigDescriptionBuildItem();
+		p1.setPropertyName("quarkus.http.cors");
+		properties.add(p1);
+		ExtendedConfigDescriptionBuildItem p2 = new ExtendedConfigDescriptionBuildItem();
+		p2.setPropertyName("quarkus.application.name");
+		properties.add(p2);
+
+		projectInfo.setProperties(properties);
+
+		testCompletionFor(value, true, null, 2, projectInfo);

```

Comment:
could you add please quarkus properties c("quarkus.http.cors",, ..

Reply:
I've made the change

Reply:
Test are not very good, thank's!